src/com/android/calendar/month/MonthWeekEventsView.java

```diff
@@ -103,9 +106,12 @@
     private static int EVENT_BOTTOM_PADDING = 1;
     private static int TODAY_HIGHLIGHT_WIDTH = 2;
     private static int SPACING_WEEK_NUMBER = 0;
+    private static int BORDER_SPACE;
```

Comment:
I think these should be lower case (as they are not constants), see: https://source.android.com/setup/contribute/code-style#follow-field-naming-conventions

(also it's not consistent in the file).

Reply:
fixed